packages/svelte/src/lib/useForm.ts

```diff
@@ -96,38 +93,31 @@ export default function useForm<TForm extends Record<string, unknown>>(
     },
     defaults(fieldOrFields?: keyof TForm | Record<keyof TForm, string>, maybeValue?: string) {
       if (typeof fieldOrFields === 'undefined') {
-        // @ts-ignore
         defaults = Object.assign(defaults, cloneDeep(this.data()))
 
         return this
       }
 
-      // @ts-ignore
       defaults = Object.assign(
-        // @ts-ignore
         cloneDeep(defaults),
         cloneDeep(typeof fieldOrFields === 'string' ? { [fieldOrFields]: maybeValue } : fieldOrFields),
       )
 
       return this
     },
     reset(...fields) {
-      // @ts-ignore
-      const resolvedData = typeof data === 'object' ? cloneDeep(defaults) : cloneDeep(data())
+      const resolvedData = typeof data === 'object' ? cloneDeep(defaults) : cloneDeep(data!())
       const clonedData = cloneDeep(resolvedData)
       if (fields.length === 0) {
-        // @ts-ignore
         this.setStore(clonedData)
       } else {
         this.setStore(
-          // @ts-ignore
```

Comment:
I'm still getting some TS errors when the `ts-ignore` is removed, are you getting those as well?

Reply:
![Capture d’écran, le 2024-06-26 à 17 02 14](https://github.com/inertiajs/inertia/assets/490234/b33ef888-ea99-4468-9ba2-5a823e8a9829)

No I don't have any kind of errors. Maybe you have out of dates nodes_modules this can happen easily

From the inertia folder root
```
find . -name 'node_modules' -type d -prune -exec rm -rf '{}' +
find . -name 'dist' -type d -prune -exec rm -rf '{}' +
npm install
cd packages/core && npm run build
cd ../svelte && npm run build
```

Restart your IDE (VSCode or other) and I believe you should be fine too?

I will add extract types just in case

Reply:
Ok, checked it out fresh and looks ok, not sure what was up. Thanks!